docs/design/slice-validation.md

```
@@ -0,0 +1,121 @@
+    epics = _validate_list(_read_json(epics_path), "epics", ["id"])
+    stories = _validate_list(_read_json(stories_path), "stories", ["id", "epic_id"])
+    tasks = _validate_list(_read_json(tasks_path), "tasks", ["id", "story_id"])
+    ...
+```
+
+### 3. Update `slice_cli` error handling
+
+The existing `except (json.JSONDecodeError, KeyError, ValueError)` block already
+catches `ValueError`, so no CLI changes are needed. The user will now see:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)